optimum/intel/utils/import_utils.py

```diff
@@ -74,6 +74,21 @@
 if _openvino_available:
     try:
         from openvino.runtime import get_version
+        import openvino_telemetry as tm
+
+        tm = tm.Telemetry(tid="G-W5E9RNLD4H",
+                          app_name="Optimum Intel",
+                          app_version=get_version(),
+                          backend='ga4',
+                          enable_opt_in_dialog=False,
+                          disable_in_ci=True)
+
+        # temporal code to check that "CI" variable is set in CI pipelines,
+        # needs to be removed before merge
+        import os
+        assert "CI" in os.environ and os.environ["CI"].lower() == "true"
+
+        tm.send_event("optimum_intel", "openvino_imported", "OpenVino {}, Optimum {}.".format(get_version(), importlib_metadata.version("optimum")))

```

Comment:
Not sure this is something we want to add in optimum

Reply:
There was a request to add Telemetry from Gorbachev Yury so we could track the number of users in optimum-intel.

Reply:
@echarlaix, any concerns about this request? We track OpenVINO usage on a regular basis and we would like to know how much inflow comes from HF ecosystem.

Reply:
@echarlaix, can you please take a look again? As @popovaan mentioned, openvino-telemetry is installed always when openvino is installed in the environment. We are just enabling it here to track optimum usage. CI jobs are excluded.